src/BreakpointViewer.cpp

```diff
@@ -395,21 +397,21 @@ void BreakpointViewer::changeTableItem(BreakpointRef::Type type, QTableWidgetIte
 			}
 
 			if (auto range = parseLocationField(index, type, item->text(), combo ? combo->currentText() : "")) {
```

Comment:
And then what happens when the break point is no longer valid because of parsing errors and the table widget loses focus? Should it be deleted from the emulator and turned into a temporary debugger object in the table widget?

Reply:
To be honest I don't really know. And I personally don't use the debugger often enough to have a strong opinion. I only wanted to bring up this point, but I'll let you decide how to best handle it. Not necessarily part of this PR.

How are incomplete breakpoint rows handled? I mean, when in the GUI you create a new row in the table, but e.g. you haven't filled in a valid address yet. It seems you cannot enable such rows.

Is this what you call a 'temporary debugger object'? If so, then my answer would be: Yes, on parse errors, instead of reverting to the previous valid value, the row should revert to a 'temporary'. But again, I don't enough experience with the debugger to really evaluate if this is really more user friendly behavior. It might also help to give some feedback on why the row is invalid, e.g. color the invalid fields in a different color (red text?)? And possibly use a gray background color for all fields in a disabled row?

No need to change anything in this PR. I'll let you decide what is best.